dbt/adapters/glue/gluedbapi/connection.py

```diff
@@ -45,14 +45,14 @@ def __init__(self, credentials: GlueCredentials, session_id_suffix: str = None,
 
     def _connect(self):
         logger.debug("GlueConnection connect called")
-        if not self.session_id:
```

Comment:
This change is not necessary. If `self.session_id` is `None`, `not self.session_id` returns `True`.